### PR TITLE
Improve logging 

### PR DESCRIPTION
**Pull request by robertkiel**
Changes:
- improve logging to make debugging connection issues and lack of addresses easier

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -131,7 +132,11 @@ export async function dialHelper(
     dhtResponse = await (libp2p._dht as any).findPeer(destination, { timeout: DEFAULT_DHT_QUERY_TIMEOUT })
   } catch (err) {
     error(
-      `Querying the DHT as peer ${libp2p.peerId.toB58String()} for ${destination.toB58String()} failed. ${err.message}`
+      `Querying the DHT for ${destination.toB58String()} failed. Knwon addresses:\n  ${(
```
Comment (andreykiryushkin) [inline src/utils/index.ts:135]: Typo

File: src/utils/index.ts [hunk 2]
```diff
@@ -131,7 +132,11 @@ export async function dialHelper(
...
+        libp2p.peerStore.get(destination)?.addresses ?? []
+      )
+        .map((addr: any) => addr.multiaddr.toString())
+        .join('\n  ')}.\n${err.message}`
```
Comment (andreykiryushkin) [inline src/utils/index.ts:139]: This line is not really readable

File: src/utils/index.ts [hunk 3]
```diff
@@ -151,9 +156,17 @@ export async function dialHelper(
...
   } catch (err) {
-    error(`Using new addresses after querying the DHT did not lead to a connection. Cannot connect. ${err.message}`)
+    error(
+      `Cannot connect to ${green(
+        destination.toB58String()
+      )}. New addresses after DHT request did not lead to a connection. Used addresses:\n  ${(
+        libp2p.peerStore.get(destination)?.addresses ?? []
+      )
+        .map((addr: any) => addr.multiaddr.toString())
+        .join('\n  ')}\n${err.message}}\n${err.message}`
```
Comment (andreykiryushkin) [inline src/utils/index.ts:168]: same with this line, why twice?

Comment (robertkiel) [reply src/utils/index.ts:168]: you're right going to refactor both

File: src/utils/index.ts (hunk 1 above)
Comment (robertkiel) [reply src/utils/index.ts]: done

**Review by andreykiryushkin — APPROVED** [review]

**Pull request merged**